### PR TITLE
add typescript typings

### PR DESCRIPTION
**Pull request by bcherny**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,10 +7,11 @@
     "sniff",
     "detection"
   ],
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (ded) [inline package.json]: please avoid bumping the version yourself.

File: package.json [hunk 2]
```diff
@@ -7,10 +7,11 @@
...
   "homepage": "https://github.com/ded/bowser",
   "author": "Dustin Diaz <dustin@dustindiaz.com> (http://dustindiaz.com)",
   "main": "./bowser.js",
+  "typings": "./typings.d.ts",
```
Comment (ded) [inline package.json]: 👍

File: typings.d.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+export const firefoxos: boolean
+export const linux: boolean
+export const mac: boolean
+export const touchpad: boolean
+export const windows: boolean
+export const windowsphone: boolean
+
+export function test(browserList: Flag[]): boolean
+
+export type Flag = "a" | "b" | "c" | "android" | "bada" | "blackberry" | "chrome" | "firefox" | "gecko" | "ios" | "msie" | "msedge" | "opera" | "phantom" | "safari" | "sailfish" | "seamonkey" | "silk" | "tizen" | "webkit" | "webos" | "chromeos" | "iphone" | "ipad" | "ipod" | "firefoxos" | "linux" | "mac" | "touchpad" | "windows" | "windowsphone"
```
Comment (ded) [inline typings.d.ts]: nitpick — add a new line

**Comment by bcherny** [comment]
@ded done

**Comment by lancedikson** [comment]
@bcherny, have you tested that? I don't use TS and ain't sure that it works :)

**Comment by bcherny** [comment]
Works great :)

Usage is:

```ts
import * as bowser from 'bowser'
const isChrome: boolean = bowser.chrome
...
```

**Comment by lancedikson** [comment]
@bcherny, ok, thanks. I'll merge it in tomorrow hotfix release.

**Pull request merged**